src/main/java/org/apache/commons/collections4/list/AbstractLinkedList.java

```diff
@@ -514,6 +513,8 @@ protected void addNodeAfter(final Node<E> node, final E value) {
      * @throws NullPointerException if either node is null
      */
     protected void addNode(final Node<E> nodeToInsert, final Node<E> insertBeforeNode) {
+        Objects.requireNonNull(nodeToInsert, "The new node on insert must not be null.");
```

Comment:
s/new node on insert/new node to insert? Like the `@param` in the javadoc above.